src/cp.js

```diff
@@ -159,7 +159,7 @@ function _cp(options, sources, dest) {
     }
   }
 
-  sources = common.expand(sources);
+  sources = common.expand(sources, {dot: true});
```

Comment:
This is still necessary because this is how we turn a directory into a list of its contents. We can probably do an alternative approach, though